rebar.config.erl

```diff
@@ -184,13 +184,17 @@ relx(Vsn, RelType, PkgType, Edition) ->
     , {overlay, relx_overlay(RelType, Edition)}
     , {overlay_vars, [ {built_on_arch, rebar_utils:get_arch()}
                      , {emqx_description, emqx_description(RelType, Edition)}
+                     , {emqx_schema_mod, emqx_schema_mod(Edition)}
```

Comment:
Just a reminder that this'll need to be added to `mix.exs` as well, in the [`template_vars/4`](https://github.com/savonarola/emqx/blob/49319faa9faaf92d784a2def063ee196d959c416/mix.exs#L412-L460) function.

Reply:
Yes, it's just the first working draft to see which checks fail :)
There will bee a bunch of cleanup fixes.